backend/app/api/v1/commons/constants.py

```diff
@@ -163,3 +164,4 @@
 # These products aren't listed by their exact names like "hce" and "ocm",
 # but instead represent grouped categories such as "Developer", "Insights", etc.
 GENERAL_PRODUCTS = ["hce", "ocm"]
+TELCO_STATUS_MAP = {"success": "passed", "failed": "failed", "failure": "failure"}
```

Comment:
So `success` means `passed`, but `failed` means `failed` and `failure` means `failure`? Why is `success` being altered, but not the others?

I'm confused. Maybe a list of the actual `status` values you've observed in Telco data would help.

Reply:
why is `success` a key instead of a value? I thought we needed to transform their status values to `success` or other?

Reply:
In this case, I'm , doing reverse mapping. So when the client sends `jobStatus` as  `success` or `failure`, I'm mapping that back to internal values like "passed", "failed", or "failure" accordingly.